zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterProcess.java

```diff
@@ -254,6 +254,12 @@ public int getNumIdleClient() {
     }
   }
 
+  public void setMaxPoolSize(int size) {
```

Comment:
The default pool size seems to be 8.
But we are configuring maxconcurrency as 10 in RemoteScheduler.
I dont think we will be able to run more than 8 paras in parallel.
Setting the pool size and scheduler max concurerency from ConfVars variable, which is set to 10.